Resources/Prototypes/_Shitmed/Entities/Structures/Machines/autodoc.yml

```
@@ -0,0 +1,102 @@
+# Autodocs
+
+- type: entity
+  parent: [ConstructibleMachine, BaseMachinePowered]
+  id: Autodoc
+  name: Autodoc Mk.XIV
+  description: A programmable robotic surgeon capable of automatically operating on patients.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)